### ISSUE TITLE
Is it possible to repeat f/F/t/T with `;,`?

### ISSUE THREAD

**Issue by megalithic** [ggandor/flit.nvim#2]
Loving flit, however, I'm wondering if there is a way to override repeats with f/t and use the original vim maps for that (or make a way to configure it)?

Thanks much!!

**Comment by ggandor** [comment]
Not in a trivial way, but I'm currently working on a complete rewrite, so stay tuned.

**Comment by tarsobcaldas** [comment]
@ggandor I can repeat with `;,`, but only right after searching with f. Is it possible to repeat the last search whenever `;,` is pressed?

**Comment by farzadmf** [comment]
Anyone found a way/an alternative to have `; ,` properly working? 🤔

**Comment by murtaza64** [comment]
@ggandor any updates here? Loving your plugins but this is a minor annoyance I've noticed lately. I'm not sure how your plugin is structured but it seems like `leap` already has this functionality with `repeat_keys` right?

**Comment by murtaza64** [comment]
Based on a quick glance through the code for `leap`, it looks like we don't save repeat state if the user provides the targets manually (https://github.com/ggandor/leap.nvim/blob/9f7cf0a87ca2c3f47df37ef6787d34a65604d248/fnl/leap/main.fnl#L722)

Why does it have to be this way?

**Comment by ggandor** [comment]
> it seems like leap already has this functionality with repeat_keys right?

Yep, but it's for the default leap motion, the saved state is assumed to contain a 2-character pattern.

> Why does it have to be this way?

We don't want to overwrite the repeat state with every custom call (e.g. treesitter selection).

TLDR, it's not a one-line tweak.

**Comment by murtaza64** [comment]
Flit is almost perfect, so I would just say that I would love this feature (and I'm happy to take a stab at implementing it if you would accept a PR)
